radix-engine-common/src/types/entity_type.rs

```diff
@@ -24,7 +24,7 @@ pub enum EntityType {
     GlobalConsensusManager = 0b10000110, //-------------- 10000 => s, 110xx => ce6m [se vanity prefix]
 
     /// A global validator entity (130 in decimal). Gives Bech32 prefix: `s` followed by one of `v`, `d`, `w` or `0`.
-    GlobalValidator = 0b10000010, //----------------- 10000 => s, 110xx => vdw0
+    GlobalValidator = 0b10000011, //--------------------- 10000 => s, 110xx => vdw0
```

Comment:
Comment no longer valid
_131 in decimal_
_011xx => vdw0_

Reply:
Nice spot. will update!